pkg/placement/raft/logger.go

```diff
@@ -86,6 +86,10 @@ func (l *loggerAdapter) ResetNamed(name string) hclog.Logger { return l }
 
 func (l *loggerAdapter) SetLevel(level hclog.Level) {}
```

Comment:
should dapr debug level log also affect this?

Reply:
It's a nop